staging/src/k8s.io/apimachinery/pkg/apimachinery/registered/registered.go

```diff
@@ -222,6 +222,8 @@ func (m *APIRegistrationManager) GroupOrDie(group string) *apimachinery.GroupMet
 	groupMeta, found := m.groupMetaMap[group]
 	if !found {
 		if group == "" {
+			// If a test triggers this panic, assure that the test's package
+			// imports "k8s.io/kubernetes/pkg/api/testapi" to run its init.
```

Comment:
@caesarxuchao I'd really like to see this package removed/contained.  We have the ability to have separate type registries now, seems like tests that need it should be constructing those.